phpBB/adm/style/admin.css

```diff
@@ -1704,6 +1704,11 @@ fieldset.submit-buttons legend {
 	height: 40px;
 }
 
+.mention-media-avatar img { /* TODO: reconsider for the future design */
```

Comment:
you should say why

Reply:
Added lines stating `with 1:1 ratio images`.